<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Users</class>
 <widget class="QDialog" name="Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size :20px;

}
 QTabWidget{
background:#333;
border-radius:15px;
}

QLabel {
postion:center;
color :#CD5C5C;
}
QTableView {
background:white;
border-radius:15px;
font-size :20px;

}
QPushButton {
background: #CD5C5C;border:none;
color:white;
 border-radius: 15px;
}
QPushButton:hover
{ color:white;
 border-radius: 15px;
background-color :#F08080;border:none;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>1101</width>
     <height>621</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste des materiels&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Liste des employés</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service employés</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher un utilisateur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Avec</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmd">
     <property name="geometry">
      <rect>
       <x>882</x>
       <y>190</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>901</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*QTableView::item
{
anchors.fill: racesWindow
        width: 840
        height: 400
       
}
*/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_line_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Ajouter un employé</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ID employé :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom employé :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom employe :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenon">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>270</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit_3">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>510</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>PassWord :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="poste">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>510</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste employé :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>200</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Modifier un employé</string>
    </attribute>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service employé</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ID employé</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom employé</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenon_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>310</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom employé</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login employé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit_6">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>480</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Password employé</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="poste_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>500</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color: #717072;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste employé</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Supprimer un employé</string>
    </attribute>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrer le code du materiel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Submit_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>330</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color :#CD5C5C;border:none;
color:white;
border-radius:15px;
}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Bienvenue à notre service materiel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
